pkg/kubelet/metrics/metrics.go

```diff
@@ -202,7 +202,7 @@ var (
 			Subsystem:      KubeletSubsystem,
 			Name:           RuntimeOperationsDurationKey,
 			Help:           "Duration in seconds of runtime operations. Broken down by operation type.",
-			Buckets:        metrics.DefBuckets,
+			Buckets:        []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 60, 300, 600, 900, 1200},
```

Comment:
We could define a variable for this custom bucket.

Reply:
Sure. Where would that variable live, here or in the `metrics` package?